test/fixers/test_dprint_fixer_callback.vader

```diff
@@ -0,0 +1,28 @@
+Before:
+  call ale#assert#SetUpFixerTest('typescript', 'dprint')
+
+After:
+  Restore
+  call ale#assert#TearDownFixerTest()
+
+Execute(The dprint callback should return the correct default values):
+  AssertFixer
+  \ {
+  \   'read_temporary_file': 1,
+  \   'command':
+  \     ale#Escape('dprint')
+  \     . ' ' . ale#Escape('')
+  \     . ' %t',
+  \ }
```

Comment:
Make sure this dictionary matches exactly what the fale#fixers#dprint#Fix() method returns:

```
    \ {
    \   'command': ale#Escape('dprint') . ' fmt --stdin %s',
    \ }
```